package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "balancetetris",
+  "version": "1.0.0",
+  "description": "<div>     <p align=\"center\">     <a href=\"https://github.com/WoochanLee/BalanceTetris\">         <img src=\"https://github.com/WoochanLee/BalanceTetris/blob/master/res/young.png?raw=true\" height=\"300\" />         <img src=\"https://github.com/WoochanLee/BalanceTetris/blob/master/res/chan.png?raw=true\" height=\"300\" />     </a>     </p>     <br>     <p align=\"center\">         <img src=\"https://github.com/WoochanLee/BalanceTetris/blob/master/res/logo.png?raw=true\" width=\"600\"/>     </p>     <br>     <br>     <p align=\"center\">🕹️You Can Play Alone👇</p>     <div align=\"center\">         <a href=\"https://woochanlee.github.io/BalanceTetris/\">             <u>https://woochanlee.github.io/BalanceTetris/</u>         </a>     </div>     <br>     <br>     <p align=\"center\">         <img src=\"https://img.shields.io/badge/license-GPL%20v3-red.svg\">         <img src=\"https://img.shields.io/badge/release-alpha-blue.svg\">     <br /> </div> <br> <br>",
```

Comment:
Minor opinion:
How about making here simpler.
Like 'a Tetris on the web platform made by ~'